index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header class="navLogo">
+      <img src="./images/navlogo.png" alt="" />
+      <img class="navHam" src="./images/hamburger.png" alt="" />
+    </header>
+    <div class="headline">
+      <h1>Charlie Giorgania</h1>
+      <p class="description">
+        Hello! I am a software developer! I can help you build a product,<br />
+        feature or website. Taka a look of my works. If you like what you see<br />
+        and have a project you need coded, don’t hesitate and contact me.
+      </p>
+
+      <button class="despButton" type="submit">Start collaboration</button>
+    </div>
+  </body>
+</html>
```

Comment:
Your application is not matching the Figma template you choose. Kindly update it according to the Figma design. If you notice, your content is not aligned properly and also there is some white space on the right side. Kindly fix this also. Also, use the same margin and padding as in the Figma design.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/176697778-96ad14c7-fe8a-4dbf-8815-a3d71b5f9bb3.png) | ![image](https://user-images.githubusercontent.com/33268927/176697685-431c4311-61e6-4057-ad42-fcec78805fba.png) |

Reply:
Fixed the styling with the figma file and added the remaining image for the "Mobile Version" of the portfolio

Reply:
Fixed it as figma file replicate